From 3d6bb7431938a0fa47346370951e306028819752 Mon Sep 17 00:00:00 2001
From: Tsuyoshi Ozawa <ozawa@apache.org>
Date: Thu, 5 Mar 2015 14:12:47 +0900
Subject: [PATCH 1353/2848] MAPREDUCE-6136. MRAppMaster doesn't shutdown file
 systems. Contributed by Brahma Reddy Battula.

(cherry picked from commit 348208014b2023894dc416dcd733a9ccdafc6e25)

Change-Id: I2ffebe4d380f1e9a6bcae3dfed8ab156d8ddfcd6
---
 .../hadoop/mapreduce/v2/app/MRAppMaster.java       |    4 ----
 1 file changed, 4 deletions(-)

diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/src/main/java/org/apache/hadoop/mapreduce/v2/app/MRAppMaster.java b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/src/main/java/org/apache/hadoop/mapreduce/v2/app/MRAppMaster.java
index b9d8818..4199efa 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/src/main/java/org/apache/hadoop/mapreduce/v2/app/MRAppMaster.java
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/src/main/java/org/apache/hadoop/mapreduce/v2/app/MRAppMaster.java
@@ -1466,10 +1466,6 @@ public static void main(String[] args) {
       String jobUserName = System
           .getenv(ApplicationConstants.Environment.USER.name());
       conf.set(MRJobConfig.USER_NAME, jobUserName);
-      // Do not automatically close FileSystem objects so that in case of
-      // SIGTERM I have a chance to write out the job history. I'll be closing
-      // the objects myself.
-      conf.setBoolean("fs.automatic.close", false);
       initAndStartAppMaster(appMaster, conf, jobUserName);
     } catch (Throwable t) {
       LOG.fatal("Error starting MRAppMaster", t);
-- 
1.7.9.5

